libpod/runtime.go

```diff
@@ -322,6 +322,14 @@ func makeRuntime(runtime *Runtime) (retErr error) {
 		}
 	}
 
+	// Create the tmpDir
+	if err := os.MkdirAll(runtime.config.Engine.TmpDir, 0751); err != nil {
+		// The directory is allowed to exist
+		if !errors.Is(err, os.ErrExist) {
```

Comment:
This check is not required, os.MkdirAll() will never return os.ErrExist.

Reply:
I just moved this code, but sure, i can remove this check.